### PR TITLE
Mh/play playable/attachments pass

### PR DESCRIPTION
**Pull request by bleehu**
Forgive the monument of slight changes, please. We've done a poor job of actually explaining how our attachment system works up until now.

### CODE REVIEW

File: 01_01_Basic_Rules.txt [hunk 1]
```diff
@@ -566,10 +566,53 @@ instead of one.
...
+These special parts are called "Weapon Attachments." You can purchase weapon
+attachments at the start of the game when building your characater, or in most
+of the trade stations and gun shops in large towns. When you buy an attachment
+from a shop, they can put the attachment on for no extra cost. While at a gun
+shop or kiosk, you can apply upgrades and attachments yourself as well. When 
+applying upgrades or customizations at a kiosk or shop, no roll is required.
+
+	Alternatively, you can attempt to add or remove a weapon attachment in the
+field. Doing so will require a skill check of DC 60 with the weapon skill that
+matches your weapon type. If you succeed, you manage to make the adjustement
```
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt]: *adjustment

File: Items/04_00_WeaponAttachments.txt [hunk 2]
```diff
@@ -201,39 +191,73 @@ following.
...
+    a pair of baffled and filtered microphones that detect heartbeats and 
+determine the approximate distance and direction to the heartbeat using clever
+techniques like phase shift analysis and parallax effect detection. 
+
+    Range Bracket Miss Chance: -1 when firing at organic targets behind cover 
+within 20m
 
 ==============================
-Barrel Attachments
+Muzzel Attachments
```
Comment (rosebrallier) [inline Items/04_00_WeaponAttachments.txt]: *Muzzle

File: Items/04_00_WeaponAttachments.txt [hunk 3]
```diff
@@ -474,45 +508,58 @@ This allows three point slings to be used without getting caught on your gear.
...
 
 == ED Electric Discharge Magazine  $2x + 20 magazine cost ==
-    1/2 the bullet's base damage is converted to electrical damage.
+
+    ED rounds contain what basically amounts to a capacitor and electrolytic
+chaff. When the round hits, it shocks whatever it hits. But this chaff does 
+little to fragment the way that conventional bullets do. 
+
+Base Damage: -50%
+Electric Damage: +50%
```
Comment (rosebrallier) [inline Items/04_00_WeaponAttachments.txt]: So, +50% Electric Damage reads like it only works if your gun already deals Electric damage (if your gun deals zero electric damage base, +50% is still zero). We want to specify that the +50% is derived from the Base Damage.

**Review by rosebrallier — APPROVED** [review]
A few spelling and one clarity note.
By the by, were we swapping Miss Chance with Accuracy? If it's not too scope-creep-ey, I'd say bite the bullet and do that here.

**Comment by Turtlelord26** [comment]
I think the rules, as they currently stand, support Accuracy rolls for attacks and Accuracy modifiers to those rolls, but individual weapons still have Miss Chances that you have to roll over with your Accuracy.

**Comment by bleehu** [comment]
How's that?

**Comment by rosebrallier** [comment]
Thank!!!

**Review by rosebrallier — APPROVED** [review]
Shipit!

File: 01_01_Basic_Rules.txt [hunk 4]
```diff
@@ -566,10 +566,53 @@ instead of one.
...
 ==============================
 
-	Weapon customization can be done anytime you are in a town, space station, or 
-other mercantile area. Ask the GM if there are weapon shops or kiosks around, 
-they should point you to one. From there you can apply upgrades and attachments.
-
-	The GM may tell you specific attachments that are available, or they may say 
-you can just go about your usual customization. See the Attachments document
-for more details on the kinds of attachments you can use, and their restrictions.
+	Almost all weapons come with peices that can be added or swapped out for
```
Comment (Turtlelord26) [inline 01_01_Basic_Rules.txt]: sp: "Pieces"

File: 01_01_Basic_Rules.txt [hunk 5]
```diff
@@ -566,10 +566,53 @@ instead of one.
...
+
+== Total Conversions ==
+
+    Total conversions only work with the specified weapon. You may only utilize 
+one weapon conversion at a time. HOWEVER, for conversions relying on switching 
+out a receiver, you may carry (at max) one additional receiver in order to swap 
+kits on the go. You must pass 2 Weapons use / knowledge [Weapons] checks in 
+order to successfully convert your weapon on the go. DC 60. Failure on one check 
+instantly jams the weapon on first shot. Failure on both checks requires an 
+additional check.
```
Comment (Turtlelord26) [inline 01_01_Basic_Rules.txt:618]: I'm not sure I understand what the "additional check" after failing two is for.

File: Items/04_00_WeaponAttachments.txt [hunk 6]
```diff
@@ -1,31 +1,34 @@
...
 Weapon Attachments
 ==============================
-    These add-ons are stat affected items or items that will add functionality 
-to a weapon. Most Attachments can be affixed to any weapon unless otherwise 
-noted. Ask a GM if you are not sure.
 
-    Please note that unless otherwise specified, only one attachment of a given 
-type may be used. For instance, one cannot use two red lasers to get a -4 Miss 
-Chance, nor can you use both a reflex sight and a 3.2x scope. 
+    These add-ons are stat affected items or items that will add functionality
```
Comment (Turtlelord26) [inline Items/04_00_WeaponAttachments.txt]: sp: "stat-affecting" ?

File: Items/04_00_WeaponAttachments.txt [hunk 7]
```diff
@@ -1,31 +1,34 @@
...
+your character looks through to see where their shots will go when they squeeze
+the trigger.
 
 ===== Level 1 Requirement =====
+
 == Tritium Glow Iron Sights  $150 ==
-    1x magnification, but allows for much better target acquisition. Can be 
+
+    A metalic notch on the rear of the rail with a small metal post toward the
+muzzle which are a simple way of lining up your shots. The metal sights have a
```
Comment (Turtlelord26) [inline Items/04_00_WeaponAttachments.txt]: sp: "muzzle, which" or "muzzle that"

File: Items/04_00_WeaponAttachments.txt [hunk 8]
```diff
@@ -201,39 +191,73 @@ following.
...
     Hip Fire Miss Chance Penalty: -2 out to 25m
 
 == Green Laser  $350 ==
+    
+    A small laser pointer that clips under the barrel of your gun and has been
+adjusted to roughly show where bullets go when you squeeze the trigger.
+
     Hip Fire Miss Chance Penalty: -1 out to 80m
 
+== Range Finder  $740 ==
```
Comment (Turtlelord26) [inline Items/04_00_WeaponAttachments.txt:223]: 🐢 approves this rail swap.

File: Items/04_00_WeaponAttachments.txt [hunk 9]
```diff
@@ -474,45 +508,58 @@ This allows three point slings to be used without getting caught on your gear.
...
 magazine/clip-based weapons. Revolver systems or rocket launchers, for example, 
 are excluded.
 
 == Hollow Point Magazine  $1.5x + 10 mag cost ==
-    +30% base damage, -1 Armor Piercing Level
+
+    Hollow point ammunition are bullets that have either a hollow tip or a tip
+with water inside. This makes the bullet fragment more dramatically on impact,
+which means that it penetrates armor less effectively. However, if it hits 
+something squishy like fruit, a person's body or the delecate silicone and boron
```
Comment (Turtlelord26) [inline Items/04_00_WeaponAttachments.txt]: sp: "delicate"

File: Items/04_00_WeaponAttachments.txt [hunk 10]
```diff
@@ -474,45 +508,58 @@ This allows three point slings to be used without getting caught on your gear.
...
 
 == ED Electric Discharge Magazine  $2x + 20 magazine cost ==
-    1/2 the bullet's base damage is converted to electrical damage.
+
+    ED rounds contain what basically amounts to a capacitor and electrolytic
+chaff. When the round hits, it shocks whatever it hits. But this chaff does 
+little to fragment the way that conventional bullets do. 
+
+Base Damage: -50%
+Electric Damage: +50% of base damage
```
Comment (Turtlelord26) [inline Items/04_00_WeaponAttachments.txt]: If my base damage is 100, then my new Base damage is 50.
But is my electric damage 50 or 25? Is it calculated from the old or the new Base?

File: Items/04_00_WeaponAttachments.txt [hunk 11]
```diff
@@ -538,32 +585,35 @@ against the poison. Poison damage is taken at the end of the turn of the
...
+    Ranged Miss Chance:  -0.5
 
 == High Velocity Ammunition  2.2x mag cost ==
-Longest Range Bracket:
-*  +10%m or +5m (whichever is higher)
 
-Base Damage:
-*  +10% (drop the remainder)
+    High Velocity Ammo uses slightly more dense metal and extra propellant to
+make bullets that jut go faster. This helps the ammo fly straighter and impart
```
Comment (Turtlelord26) [inline Items/04_00_WeaponAttachments.txt]: sp: "just"

**Review by Turtlelord26 — APPROVED** [review]
Is that ... is that real live prose for individual attachments? 

It's so much better 👍 👍 

Mostly nb and sp inlines. Fairly serious clarity issue at 562 that I would like addressed but might be scope creepy.

File: Items/04_00_WeaponAttachments.txt (hunk 9 above)
Comment (bleehu) [reply Items/04_00_WeaponAttachments.txt]: I swear in front of my Lord God Most High that I did at some point attend an English language class in public school!

File: Items/04_00_WeaponAttachments.txt (hunk 11 above)
Comment (bleehu) [reply Items/04_00_WeaponAttachments.txt]: *embarrassed flopping on the ground and ugly-crying*

File: 01_01_Basic_Rules.txt (hunk 5 above)
Comment (bleehu) [reply 01_01_Basic_Rules.txt:618]: These rules where written by Trowl in the Time Before Time-and-source-control. I'm assuming that if you botch both weapon mod rolls, your character knows they've screwed up and have to do the modification all over again.

**Comment by bleehu** [comment]
Made grammar fixes and merged.

File: Items/04_00_WeaponAttachments.txt [hunk 12]
```diff
@@ -558,7 +558,7 @@ Base Damage: -10%
 chaff. When the round hits, it shocks whatever it hits. But this chaff does 
 little to fragment the way that conventional bullets do. 
 
-Base Damage: -50%
+Balistic Damage: -50% of base damage
```
Comment (Turtlelord26) [inline Items/04_00_WeaponAttachments.txt]: Oh nooo. You got so close, and yet so far....

sp: Ballistic

**Pull request merged**